src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/HyperlinkType.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.PackageManagement.Telemetry
+{
+    public enum HyperlinkType
+    {
+        Header,
+        License,
```

Comment:
Not all hyperlinks are tracked. A follow up issue is filed to complete tracking of all URIs. See https://github.com/NuGet/Client.Engineering/issues/1124